.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
Please don't add a .gitignore for user specific stuff (e.g. the name of your build folder). You can use `.git/info/exclude` instead, see here: https://stackoverflow.com/a/1753078

Reply:
haha I just knew it such a config, thanks anyway